### PR TITLE
doc: typo

### PR DESCRIPTION
**Pull request by DanishOnGit**
fix typo

### CODE REVIEW

**Review by vchapel — COMMENTED** [review]
"If you notice" would be better English, and should "jank" be "junk" ... or is there a better word, such as "noise" or "clutter" or "errors" ?

**Comment by DanishOnGit** [comment]
> "If you notice" would be better English, and should "jank" be "junk" ... or is there a better word, such as "noise" or "clutter" or "errors" ?

Yes "If you notice" is better. Or, "If you are noticing..." also works fine imo. Not sure about the word "jank"  though!

**Comment by vchapel** [comment]
> > "If you notice" would be better English, and should "jank" be "junk" ... or is there a better word, such as "noise" or "clutter" or "errors" ?
> 
> Yes "If you notice" is better. Or, "If you are noticing..." also works fine imo. Not sure about the word "jank" though!

Yes, both introductions are good. We do need a better word than "jank"  B^)

File: microsoft-edge/devtools-guide-chromium/rendering-tools/index.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ Take a recording in the **Performance** tool and look for suspiciously long `Eva
 <!--todo: add Recording section when available  -->  
 <!--todo: add Profile JavaScript (JS Profiler) section when available  -->  
 
-f you noticing quite a bit of jank in your JavaScript, you may need to take your analysis to the next level and collect a JavaScript CPU profile.  CPU profiles show where runtime is spent within the functions of your page.  Learn how to create CPU profiles in [Speed Up JavaScript Runtime][DevtoolsRenderingToolsJavascriptRuntime].
+If you noticing quite a bit of jank in your JavaScript, you may need to take your analysis to the next level and collect a JavaScript CPU profile.  CPU profiles show where runtime is spent within the functions of your page.  Learn how to create CPU profiles in [Speed Up JavaScript Runtime][DevtoolsRenderingToolsJavascriptRuntime].
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/rendering-tools/index.md:46]:
```suggestion
If you notice quite a bit of jank in your JavaScript, you may need to take your analysis to the next level and collect a JavaScript CPU profile.  CPU profiles show where run time is spent within the functions of your page.  Learn how to create CPU profiles in [Speed Up JavaScript Runtime][DevtoolsRenderingToolsJavascriptRuntime].
```
Not sure but also changed "runtime" to "run time" to emphasize time.  Need to check surrounding word-usage of 'runtime', 'run', and 'time', or CPU cycles, per profiling code/ function time.

**Review by mikehoffms — CHANGES_REQUESTED** [review]
An Edit pass.

File: microsoft-edge/devtools-guide-chromium/rendering-tools/index.md (hunk 1 above)
Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/rendering-tools/index.md:46]: Please click Commit suggestion (or apply changes locally, then push).  Then I'll Approve.

**Review by mikehoffms — APPROVED** [review]
An Edit pass.

**Review by vchapel — APPROVED** [review]
LGTM

File: microsoft-edge/devtools-guide-chromium/rendering-tools/index.md [hunk 2]
```diff
@@ -43,7 +43,7 @@ Take a recording in the **Performance** tool and look for suspiciously long `Eva
 <!--todo: add Recording section when available  -->  
 <!--todo: add Profile JavaScript (JS Profiler) section when available  -->  
 
-f you noticing quite a bit of jank in your JavaScript, you may need to take your analysis to the next level and collect a JavaScript CPU profile.  CPU profiles show where runtime is spent within the functions of your page.  Learn how to create CPU profiles in [Speed Up JavaScript Runtime][DevtoolsRenderingToolsJavascriptRuntime].
+If you notice quite a bit of jank in your JavaScript, you may need to take your analysis to the next level and collect a JavaScript CPU profile.  CPU profiles show where runtime is spent within the functions of your page.  Learn how to create CPU profiles in [Speed up JavaScript runtime][DevtoolsRenderingToolsJavascriptRuntime].
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/rendering-tools/index.md:46]: I will (or you could) full-text search "jank" in repo, two articles use the word in visible text, needs to be defined in both articles.  wiktionary: "jank ( uncountable ) ( computing, slang, rare) Perceptible pause in the smooth rendering of a software application 's user interface due to slow operations or poor interface design. quotations ".  I plan to enter Suggestions in two articles in this PR.

**Review by mikehoffms — CHANGES_REQUESTED** [review]
Need to define 'jank' in two articles.

File: microsoft-edge/devtools-guide-chromium/rendering-tools/index.md (hunk 2 above)
Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/rendering-tools/index.md:46]:
```suggestion
If you notice quite a bit of jank (interruptions of rendering) in your JavaScript, you may need to take your analysis to the next level and collect a JavaScript CPU profile.  CPU profiles show where runtime is spent within the functions of your page.  Learn how to create CPU profiles in [Speed up JavaScript runtime][DevtoolsRenderingToolsJavascriptRuntime].
```

**Review by mikehoffms — APPROVED** [review]

**Review by vchapel — COMMENTED** [review]
LGTM

**Pull request merged**